pkg/cmd/server/origin/run_components.go

```diff
@@ -508,3 +510,31 @@ func (c *MasterConfig) RunClusterQuotaMappingController() {
 		go c.ClusterQuotaMappingController.Run(5, utilwait.NeverStop)
 	})
 }
+
+func (c *MasterConfig) RunClusterQuotaReconciliationController() {
+	osClient, kClient := c.ResourceQuotaManagerClients()
+	resourceQuotaRegistry := quotainstall.NewRegistry(kClient)
+	groupKindsToReplenish := []unversioned.GroupKind{
+		kapi.Kind("Pod"),
+		kapi.Kind("Service"),
+		kapi.Kind("ReplicationController"),
+		kapi.Kind("PersistentVolumeClaim"),
+		kapi.Kind("Secret"),
+		kapi.Kind("ConfigMap"),
```

Comment:
I will add a utility method upstream to pull this list so we can pull from a common set?